test/integration/framework/framework.go

```diff
@@ -64,16 +69,20 @@ func (f *Framework) Setup() (context.Context, *rest.Config, client.Client) {
 	err = kueue.AddToScheme(scheme.Scheme)
 	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
 
-	//+kubebuilder:scaffold:scheme
+	// +kubebuilder:scaffold:scheme
 
 	k8sClient, err := client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
 	gomega.ExpectWithOffset(1, k8sClient).NotTo(gomega.BeNil())
 
-	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
+	mgrOpts := manager.Options{
 		Scheme:             scheme.Scheme,
 		MetricsBindAddress: "0", // disable metrics to avoid conflicts between packages.
-	})
+	}
+	if f.TweakManagerOptions != nil {
+		f.TweakManagerOptions(&mgrOpts)
+	}
```

Comment:
can we do something like this and avoid adding `GetTestEnv`

```suggestion
	if len(f.WebhookPath) > 0 {
		webhookInstallOptions := f.testEnv.WebhookInstallOptions
		mgrOpts.Host = webhookInstallOptions.LocalServingHost
		mgrOpts.Port = webhookInstallOptions.LocalServingPort
		mgrOpts.CertDir = webhookInstallOptions.LocalServingCertDir
	}
```